python/hail/docs/aggregators.rst

```diff
@@ -28,7 +28,7 @@ The ``aggregators`` module is exposed as ``hl.agg``, e.g. ``hl.agg.sum``.
     stats
     product
     fraction
-    hardy_weinberg
+    hwe_test
```

Comment:
would prefer hardy_weinberg_test

Reply:
Agreed.

Reply:
I also think we need a strong reason to rename -- these will break pipelines.

Reply:
That's why I thought it best to make these changes comprehensively, so they only break once. It should now be clear from the name and docs that `hardy_weinberg_test` is a frequentist test that returns a p-value, just like `fisher_exact_test`, `chi_sq_test`, etc, and that the function, aggregator, and variant_qc fields behave consistently (modulo missing when no bi-allelics in the variant_qc case as documented).